synapse/storage/databases/main/room_batch.py

```diff
@@ -18,7 +18,7 @@
 
 
 class RoomBatchStore(SQLBaseStore):
-    async def get_insertion_event_by_batch_id(
+    async def get_insertion_event_id_by_batch_id(
         self, room_id: str, batch_id: str
     ) -> Optional[str]:
         """Retrieve a insertion event ID.
```

Comment:
How do I get the CI running again? Seems like it hasn't been running on my commits for the past few days 🤔

Reply:
Looks like it's working now 🤷